doc/api/errors.md

```diff
@@ -238,7 +238,7 @@ function MyError() {
 
 // Without passing MyError to captureStackTrace, the MyError
 // frame would show up in the .stack property. By passing
-// the constructor, we omit that frame and all frames above it.
+// the constructor, we omit that frame, and retain all frames below it.
```

Comment:
Sorry to come late to the party, but why is this more clear?
Also it's not just `that frame` it's all the frames above it, as stated before.

Maybe "deeper than" might be more clear then "below"